README.md

```diff
@@ -1,28 +1,34 @@
-# Git Repository Template
+This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app).
 
-Project repository template to set up all public projects at [Nimble](https://nimblehq.co/)
+## Getting Started
 
-## Usage
+First, run the development server:
 
-Clone the repository
+```bash
+npm run dev
+# or
+yarn dev
+```
 
-`git clone git@github.com:nimblehq/git-template.git`
+Open [http://localhost:3000](http://localhost:3000) with your browser to see the result.
 
-## License
+You can start editing the page by modifying `pages/index.js`. The page auto-updates as you edit the file.
 
-This project is Copyright (c) 2014 and onwards. It is free software,
-and may be redistributed under the terms specified in the [LICENSE] file.
+[API routes](https://nextjs.org/docs/api-routes/introduction) can be accessed on [http://localhost:3000/api/hello](http://localhost:3000/api/hello). This endpoint can be edited in `pages/api/hello.js`.
 
-[LICENSE]: /LICENSE
+The `pages/api` directory is mapped to `/api/*`. Files in this directory are treated as [API routes](https://nextjs.org/docs/api-routes/introduction) instead of React pages.
 
-## About
+## Learn More
 
-![Nimble](https://assets.nimblehq.co/logo/dark/logo-dark-text-160.png)
+To learn more about Next.js, take a look at the following resources:
 
-This project is maintained and funded by Nimble.
+- [Next.js Documentation](https://nextjs.org/docs) - learn about Next.js features and API.
+- [Learn Next.js](https://nextjs.org/learn) - an interactive Next.js tutorial.
 
-We love open source and do our part in sharing our work with the community!
-See [our other projects][community] or [hire our team][hire] to help build your product.
+You can check out [the Next.js GitHub repository](https://github.com/vercel/next.js/) - your feedback and contributions are welcome!
 
-[community]: https://github.com/nimblehq
-[hire]: https://nimblehq.co/
```

Comment:
As this is a public repository, what about keeping the Nimble "hiring" message? :D  This way we might get to find more awesome teammates from Github ^^

Reply:
sure!
actually the readme will go through further changes when the ci/cd will be introduced hopefully, kept the hiring message!